src/patternfly/components/Scrollspy/scrollspy.scss

```diff
@@ -0,0 +1,168 @@
+  --pf-c-scrollspy__list--before--BorderColor: var(--pf-global--BorderColor--100);
+  --pf-c-scrollspy__list--before--BorderTopWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-scrollspy__list--before--BorderRightWidth: 0;
+  --pf-c-scrollspy__list--before--BorderBottomWidth: 0;
+  --pf-c-scrollspy__list--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy--m-vertical__list--before--BorderLeftWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-scrollspy--m-vertical__list--before--BorderTopWidth: 0;
+
+  // with background
+  --pf-c-scrollspy--m-background--BackgroundColor: var(--pf-global--BackgroundColor--200);
+
+  // link
+  --pf-c-scrollspy__link--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__link--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__link--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__link--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__link--link--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-scrollspy__link--OutlineOffset: calc(-1 * var(--pf-global--spacer--sm));
+
+  // before
+  --pf-c-scrollspy__link--before--BorderTopWidth: 0;
+  --pf-c-scrollspy__link--before--BorderRightWidth: 0;
+  --pf-c-scrollspy__link--before--BorderBottomWidth: 0;
+  --pf-c-scrollspy__link--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy__link--before--BorderColor: transparent;
+  --pf-c-scrollspy__link--focus--before--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-scrollspy__link--focus--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy__link--focus--before--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-scrollspy__item--m-current__link--before--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-scrollspy__item--m-current__link--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy__item--m-current__link--before--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-scrollspy--m-vertical__link--focus--before--BorderTopWidth: 0;
+  --pf-c-scrollspy--m-vertical__link--focus--before--BorderLeftWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderTopWidth: 0;
+  --pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderLeftWidth: var(--pf-global--BorderWidth--lg);
+
+  // text
+  --pf-c-scrollspy__item-text--Color: var(--pf-global--Color--200);
+  --pf-c-scrollspy__link--hover__item-text--Color: var(--pf-global--Color--100);
+  --pf-c-scrollspy__link--focus__item-text--Color: var(--pf-global--Color--100);
+  --pf-c-scrollspy__item--m-current__item-text--Color: var(--pf-global--Color--100);
+
+  // label
+  --pf-c-scrollspy__label--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__label--PaddingBottom: var(--pf-global--spacer--md);
+
+  padding-right: var(--pf-c-scrollspy--PaddingRight);
+  padding-left: var(--pf-c-scrollspy--PaddingLeft);
+  background-color: var(--pf-c-scrollspy--BackgroundColor);
+
+  &.pf-m-background {
```

Comment:
Is this needed? Seems like the section it's in would define the background.

Reply:
what do you mean by the section its in would define the background ? like the list would ?

Reply:
This looks like a page/header section that has scrollspy in it, as opposed to the whole grey box being the scrollspy component.

<img width="1691" alt="Screen Shot 2020-10-21 at 12 45 05 PM" src="https://user-images.githubusercontent.com/35148959/96757630-4f866b00-139b-11eb-9112-810b72025d2b.png">

Reply:
@ajacobs21e what are your thoughts here ?

Reply:
Yeah this is an example of a band that includes a logo, the jump links component, and a button. The whole band would need to be sticky and above the content (content scrolls beneath it).

Reply:
Is that a combination of components or would the jumplinks component have a way to put other components in / on it?

Reply:
On our side, the grey box is a stick page section. Inside the grey box would be a split/flex/grid layout with the icon in left part of the layout, scrollspy in the center, and button in the right. So the scrollspy wouldn't need a grey background - it doesn't have a background and can be used in other things that have white or grey backgrounds.